docker/templates/killbill/latest/Dockerfile

```diff
@@ -27,5 +27,20 @@ ENV KPM_INSTALL_CMD ansible-playbook $ANSIBLE_OPTS  \
                                      -e tomcat_group=$TOMCAT_GROUP \
                                      -e catalina_base=$CATALINA_BASE \
                                      $KILLBILL_CLOUD_ANSIBLE_ROLES/killbill.yml
+
+ENV MIGRATIONS_CMD ansible-playbook $ANSIBLE_OPTS \
+                                    -e kpm_install_dir=$KPM_INSTALL_DIR \
+                                    -e kpm_version=$KPM_VERSION \
+                                    -e kpm_yml=$KILLBILL_INSTALL_DIR/kpm.yml \
+                                    -e kb_config_dir=$KILLBILL_INSTALL_DIR \
+                                    -e kb_plugins_dir=$KILLBILL_INSTALL_DIR/bundles \
+                                    -e tomcat_owner=$TOMCAT_OWNER \
+                                    -e tomcat_group=$TOMCAT_GROUP \
+                                    -e catalina_base=$CATALINA_BASE \
+                                    $KILLBILL_CLOUD_ANSIBLE_ROLES/migrations.yml
+
+# Install Flyway in the image
+RUN ansible-playbook $ANSIBLE_OPTS $KILLBILL_CLOUD_ANSIBLE_ROLES/flyway.yml
```

Comment:
I thought we would not run that automatically ?

Reply:
This is the `flyway.yml` playbook, not the migrations one: this will simply install our Flyway binary in the image but not run anything.